OTAcceleratorCore/OTMultiPartyCommunicator.m

```diff
@@ -222,11 +213,23 @@ - (NSError *)connect {
 }
 
 - (NSError *)disconnect {
+    MultiPartyLoggingWrapper *loggingWrapper = [MultiPartyLoggingWrapper sharedInstance];
+    
+    [loggingWrapper.logger logEventAction:KLogActionDisconnect
+                                variation:KLogVariationAttempt
+                               completion:nil];
     
     // need to explicitly unpublish and unsubscriber if the communicator is the only accelerator to dismiss from the common session
     // when there are multiple accelerator packs, the common session will not call the disconnect method until the last delegate object is removed
     if (self.publisher) {
-        
+        [loggingWrapper.logger logEventAction:KLogActionStopPublishing
+                                    variation:KLogVariationAttempt
+                                   completion:nil];
+        if (self.screenSharingView) {
+            [loggingWrapper.logger logEventAction:KLogActionStopScreensharing
+                                        variation:KLogVariationAttempt
+                                       completion:nil];
+        }
```

Comment:
@Lucashuang0802 in Android, we are following the approach below:
- start/stop publishing means starting to share your media independently on if it is screen or not.
- start/stop screensharing means you are publishing and also, it is the screensharing case.

So the stopPublishing log should not be in the else, it is covering the ss case as well. wdyt?